src/Symfony/Component/Templating/DelegatingEngine.php

```diff
@@ -17,6 +17,8 @@
  * DelegatingEngine selects an engine for a given template.
  *
  * @author Fabien Potencier <fabien@symfony.com>
+ *
+ * @deprecated since Symfony 6.5, use Twig instead
```

Comment:
```suggestion
 * @deprecated since Symfony 6.4, use Twig instead
```

We're not planning to release a 6.5. 😉

Reply:
oops, fixed.